lib/data/wrapper.js

```diff
@@ -153,6 +153,24 @@ const data = {
         client = newClient;
         return client;
     },
+
+    checkHealth: (log, done) => {
+        client.healthcheck(log, (err, result) => {
+            const respBody = {};
+            if (err) {
+                log.warn('error from sproxyd', { error: err });
```

Comment:
I asked specifically for warn here, because the healthcheck route will be called every few seconds by the load balancer and/or monitoring agent, so it would end up spamming logs.